mgmt/RecordsConfig.cc

```diff
@@ -365,12 +365,17 @@ static const RecordElement RecordsConfig[] =
   //        # verbose via string
   //        #
   //        # 0 - no extra info added to string
-  //        # 1 - all extra information added
-  //        # 2 - some extra info added
+  //        # 1 - minimal information.
+  //        # 2 - more information
+  //        # 3 - maximum information.
   {RECT_CONFIG, "proxy.config.http.request_via_str", RECD_STRING, "ApacheTrafficServer/" PACKAGE_VERSION, RECU_DYNAMIC, RR_NULL, RECC_NULL, nullptr, RECA_NULL}
   ,
   {RECT_CONFIG, "proxy.config.http.response_via_str", RECD_STRING, "ApacheTrafficServer/" PACKAGE_VERSION, RECU_DYNAMIC, RR_NULL, RECC_NULL, nullptr, RECA_NULL}
   ,
+  {RECT_CONFIG, "proxy.config.http.request_via_transport", RECD_STRING, "compact", RECU_DYNAMIC, RR_NULL, RECC_NULL, nullptr, RECA_NULL}
+  ,
```

Comment:
It's documented. It generates decent error messages if you get the string wrong (try setting it to "zwoop" to see).